README.md

```diff
@@ -45,11 +47,13 @@ When built, the UI bundle will be available in directory `build/ui-bundle.zip`
 Assuming that your local copy of the docs-ui repository is at the same level as your local copy of the docs directory, then in your docs repository’s root directory, run the following command:
 
 ```
-antora \
-    --ui-bundle-url ../docs-ui/build/ui-bundle.zip \
-    site.yml
+yarn antora \
+    --url http://localhost:8080 \
+    --ui-bundle-url ../docs-ui/build/ui-bundle.zip 
```

Comment:
This path doesn't make sense within the ui repository?

Reply:
It makes sense. We have tested this procedure when we we started using antora.
It is important to know the text above this command. In a nutshell it tells that you run this command from docs. Doing so any change in docs-ui show the full impact on docs before that change gets merged.

Reply:
Within this repository we don't care about the docs repo, the docs repo got instructions how to build the docs with a custom theme. Within this repo we should only reference the dummy content.

Reply:
There is some truth in what you say, but just look at #37, without using the command as written in the comment, I would have never identified issues with different content resizing the browser window. You just can´t see that without including real content.
The question is, do we want to make things easier for testing and find possible issues early or just stick inside a repro where we have post merge issues.
`gulp preview` as written in readme already allows showing dummy content.
This PR just updates the existing readme to be inline with the current docs procedure. By that helping with an easy way finding possible issues early.

Reply:
I dislike that many things which are not part of the ownCloud docs itself are getting overdocumented.

Reply:
Agreed, in this case we could in the majority of the cases just add some links to man pages, config files and source code. For the ordinary user and reader we are writing expanded docs.